### DISCUSSION TITLE
v3.0.95

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#9647] [General]
## What's Changed
* add HasLifecycleHooks for CanUpdateState by @noxoua in https://github.com/filamentphp/filament/pull/9495
* Allow infolist modal to infolist action by @damiandejong in https://github.com/filamentphp/filament/pull/9563
* Update 01-getting-started.md by @NaungYeHtet in https://github.com/filamentphp/filament/pull/9584
* Traslations Croatian language by @erikroznbeker in https://github.com/filamentphp/filament/pull/9592
* Add Slovak (sk) translations by @erikgreasy in https://github.com/filamentphp/filament/pull/9587
* Update Ukrainian translations by @ivanBell199 in https://github.com/filamentphp/filament/pull/9608
* Added and updated Catalan (ca) translations by @Orphail in https://github.com/filamentphp/filament/pull/9603
* Improve make command questions by @vassilidev in https://github.com/filamentphp/filament/pull/9600
* fix for ImageColumn to accept Base64 image `data:image` by @atmonshi in https://github.com/filamentphp/filament/pull/9607
* fix(lang): missing polish translations by @mrwogu in https://github.com/filamentphp/filament/pull/9615
* Enhance Dynamic Record Title Retrieval  by @a21ns1g4ts in https://github.com/filamentphp/filament/pull/9595
* feat: implement possibilty to apply or not tenant relationship on res… by @Ashk2a in https://github.com/filamentphp/filament/pull/9404
* Support icon color for text entry and column by @aymanalareqi in https://github.com/filamentphp/filament/pull/9522
* feat: allow dynamically setting the card width on SimplePage by @ralphjsmit in https://github.com/filamentphp/filament/pull/9621
* Update Russian translations by @ivanBell199 in https://github.com/filamentphp/filament/pull/9624
* Add missing closing parenthesis in scoping code example by @prip in https://github.com/filamentphp/filament/pull/9632
* Added deleted_at to columns that should be hidden by default by @Rattone in https://github.com/filamentphp/filament/pull/9629
* Update pt_PT and pt_BR Translations by @gdsrmygdsrjr in https://github.com/filamentphp/filament/pull/9645
* Fix required validation for concealed select by @tomas-doudera in https://github.com/filamentphp/filament/pull/9616
* Update Indonesian Translations by @risangbaskoro in https://github.com/filamentphp/filament/pull/9642

## New Contributors
* @NaungYeHtet made their first contribution in https://github.com/filamentphp/filament/pull/9584
* @erikroznbeker made their first contribution in https://github.com/filamentphp/filament/pull/9592
* @erikgreasy made their first contribution in https://github.com/filamentphp/filament/pull/9587
* @ivanBell199 made their first contribution in https://github.com/filamentphp/filament/pull/9608
* @Orphail made their first contribution in https://github.com/filamentphp/filament/pull/9603
* @vassilidev made their first contribution in https://github.com/filamentphp/filament/pull/9600
* @mrwogu made their first contribution in https://github.com/filamentphp/filament/pull/9615
* @Rattone made their first contribution in https://github.com/filamentphp/filament/pull/9629
* @gdsrmygdsrjr made their first contribution in https://github.com/filamentphp/filament/pull/9645
* @risangbaskoro made their first contribution in https://github.com/filamentphp/filament/pull/9642

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.0.93...v3.0.95

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.0.95'>v3.0.95</a>.</em>

**Comment by Rattone** [comment]
Thanks for the mention and for your time
